user-guide/Advanced_Functionality/Databases/Indexing_database/OpenSearch_Database/Installing_OpenSearch_database.md

```diff
@@ -14,6 +14,8 @@ Supported versions:
 > [!NOTE]
 > OpenSearch is supported on Windows from OpenSearch 2.4 onwards.
 
+We promote the use of Ubuntu LTS as the preferred Linux distribution. The procedure mentioned below will work on any Debian-based system, including Ubuntu.
```

Comment:
To make sure people can find more detailed info, I'd rephrase this as follows with a link:

```suggestion
We recommend using the latest version of Ubuntu LTS. However, OpenSearch also supports other Linux distributions. See [Operating system compatibility](https://opensearch.org/docs/latest/install-and-configure/install-opensearch/index/#operating-system-compatibility).
```

Maybe we should also add this in the note above, instead of adding it as a simple paragraph?